From: "Javier M. Mellid" <jmunhoz@igalia.com>
Date: Tue, 26 Jul 2016 14:56:50 +0200
Subject: rgw: doc: add multipart uploads copy part feature as supported

Signed-off-by: Javier M. Mellid <jmunhoz@igalia.com>
(cherry picked from commit 8a7ebeee2ff3f10ceb23b7fa43e43c3c450efe22)
(cherry picked from commit a54a7ada65fafc88251897b2739b364bf59be128)

Resolves (with following): rhbz#1416160
---
 doc/radosgw/s3.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/radosgw/s3.rst b/doc/radosgw/s3.rst
index 67f497c..15164e3 100644
--- a/doc/radosgw/s3.rst
+++ b/doc/radosgw/s3.rst
@@ -70,7 +70,7 @@ The following table describes the support status for current Amazon S3 functiona
 +---------------------------------+-----------------+----------------------------------------+
 | **Copy Object**                 | Supported       |                                        |
 +---------------------------------+-----------------+----------------------------------------+
-| **Multipart Uploads**           | Supported       | (missing Copy Part)                    |
+| **Multipart Uploads**           | Supported       |                                        |
 +---------------------------------+-----------------+----------------------------------------+
 
 Unsupported Header Fields
